.sonarcloud.properties

```diff
@@ -1,4 +1,4 @@
 # Disable specific file since it would introduce more complexity to reduce it - mainly code complexity and complex template literals
-sonar.exclusions=apps/studio/public/js/monaco/**,apps/studio/src/tailwind.css,apps/studio/src/components/SplitPane/**,apps/studio-next/cypress/**,apps/studio-next/Dockerfile
+sonar.exclusions=apps/studio/public/js/monaco/**,apps/studio/src/tailwind.css,apps/studio/src/components/SplitPane/**,apps/studio/cypress/**,apps/studio/Dockerfile
 # Disable duplicate code in tests since it would introduce more complexity to reduce it.
-sonar.cpd.exclusions=apps/studio/**,apps/studio-next/src/components/Navigationv3.tsx,apps/studio-next/src/components/Navigation.tsx,apps/studio-next/cypress/**,apps/studio-next/src/helpers/driver.ts
+sonar.cpd.exclusions=apps/studio/**,apps/studio/src/components/Navigationv3.tsx,apps/studio/src/components/Navigation.tsx,apps/studio/cypress/**,apps/studio/src/helpers/driver.ts
```

Comment:
Remove the cpd exclusions which were not there before migration.

Reply:
ah yes, didnt realise the changes i made are redundant.